src/window_gui.h

```diff
@@ -11,7 +11,10 @@
 #define WINDOW_GUI_H
 
 #include <list>
+#include <algorithm>
```

Comment:
Not a critique, but a question: What are these? I've never seen references to these before in OpenTTD. (Remember I'm new to the development team. 🙂 )

Reply:
This is a standard C++ header, which comes with all standards-compliant C++ compiler.  See [C++ reference page for `<algorithm>`](https://en.cppreference.com/w/cpp/header/algorithm).

In this case `<algorithm>` is needed for `std::adjacent_find` and `<functional>` is needed for `std::not_fn`.

Reply:
Thanks for the explanation.